airgun/widgets.py

```diff
@@ -719,7 +719,11 @@ def fill(self, value):
         :param value: string with value that should be used for field update
             procedure
         """
-        self.edit_button.click()
+        # in some cases editing fields automatically triggers editing others,
+        # so the field may be opened for editing and clicking "edit" button is
+        # not required for it
+        if self.edit_button.is_displayed:
```

Comment:
not 100% happy with that approach as `edit_button is not displayed` doesn't equal `edit_field is displayed`, but let's resolve issue if that ever happen